src/tokens/hooks.ts

```diff
@@ -133,6 +133,14 @@ export function useTokenInfo(tokenId?: string): TokenBalance | undefined {
   return tokenId ? tokens[tokenId] : undefined
 }
 
+export function useTokensInfo([...tokenIds]: string[]): (TokenBalance | undefined)[] {
```

Comment:
```suggestion
export function useTokensInfo(tokenIds: string[]): (TokenBalance | undefined)[] {
```
can we just do this^?

Reply:
Changed in 60bb96464c22d6311184c76ff7077b999aa700fc!